client/src/App.js

```diff
@@ -2,16 +2,16 @@ import React, { Component } from "react";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
 import "./App.css";
 import { library } from "@fortawesome/fontawesome-svg-core";
-import { faTrash, faFilter, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { faTrash, faFilter, faPlus, faEllipsisH, faCaretDown, faSearch, faEdit } from "@fortawesome/free-solid-svg-icons";
 import Login from "./components/Login";
 import Header from "./components/Layout/Header";
 import Projects from "./components/Projects";
 import ProjectNew from "./components/Projects/ProjectNew";
 import ProjectEdit from "./components/Projects/ProjectEdit";
-import Members from "./components/Members";
+import Users from "./components/Members";
```

Comment:
I think it's more convenient if the component names are the same as the component folder name